biolink/api/entityset/endpoints/slimmer.py

```diff
@@ -47,7 +48,6 @@ def get(self, category):
         results = map2slim(subjects=prots,
                            slim=slim,
                            rows=200,
-                           exclude_automatic_assertions=True,
```

Comment:
per the call, should be `=False` but otherwise looks great.

Reply:
The `exclude_automatic_assertions` is `False` by default, and it is passed in to `map2slim` via `args` dictionary.

Reply:
gotcha